openforcefield/typing/engines/smirnoff/parameters.py

```diff
@@ -4353,9 +4368,24 @@ def create_force(self, system, topology, **kwargs):
                 atom_indices = (
                     charge_increment_match.environment_match.topology_atom_indices
                 )
-                charge_increment = charge_increment_match.parameter_type
+                charge_increments = copy.deepcopy(charge_increment_match.parameter_type.charge_increment)
+
+                # If we've been provided with one less charge increment value than tagged atoms, assume the last
+                # tagged atom offsets the charge of the others to make the chargeincrement net-neutral
+                if len(charge_increments) - len(atom_indices) == -1:
```

Comment:
This does the check backwards from `diff = n_tags - n_increments` above; I don't care which is used but they should be identical for readability

Reply:
Good idea. Changed in 9cc29a5.